app-android/src/main/java/com/ustadmobile/port/android/view/Login2Fragment.kt

```diff
@@ -119,5 +146,102 @@ class Login2Fragment : UstadBaseFragment(), Login2View {
         mPresenter = null
         mBinding = null
     }
+}
+
+@Composable
+private fun LoginScreen(
+    uiState: LoginUiState = LoginUiState(),
+    onClickLogin: () -> Unit = {},
+    onClickCreateAccount: () -> Unit = {},
+    onClickConnectAsGuest: () -> Unit = {},
+    onUsernameValueChange: (String) -> Unit = {},
+    onPasswordValueChange: (String) -> Unit = {},
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    )  {
+
+        UstadTextEditField(
+            value = "",
+            label = stringResource(id = R.string.username),
+            onValueChange = {
+                onUsernameValueChange(it)
+            },
+            error = uiState.usernameError,
+            enabled = true,
+        )
+
+        UstadTextEditField(
+            value = "",
+            label = stringResource(id = R.string.password),
+            onValueChange = {
+                onPasswordValueChange(it)
+            },
+            error = uiState.passwordError,
+            enabled = true,
+            trailingIcon = {
+                Icon(
+                    painter = painterResource(R.drawable.ic_baseline_visibility_24),
+                    contentDescription = "")
+            }
+        )
+
+        Spacer(modifier = Modifier.height(10.dp))
+
+        Button(
+            onClick = onClickLogin,
+            modifier = Modifier
+                .fillMaxWidth(),
+            colors = ButtonDefaults.buttonColors(
+                contentColor = Color.Black,
+                backgroundColor = colorResource(id = R.color.secondaryColor)
+            )
+        ) {
+            Text(stringResource(R.string.login).uppercase())
+        }
 
+        Spacer(modifier = Modifier.height(10.dp))
+
+        OutlinedButton(
+            onClick = onClickCreateAccount,
+            modifier = Modifier
+                .fillMaxWidth(),
+            colors = ButtonDefaults.buttonColors(
+                backgroundColor = Color.Transparent,
+                contentColor = colorResource(id = R.color.primaryColor)
+            )
+        ) {
+            Text(stringResource(R.string.create_account).uppercase())
+        }
+
+        Spacer(modifier = Modifier.height(10.dp))
+
+        OutlinedButton(
+            onClick = onClickConnectAsGuest,
+            modifier = Modifier
+                .fillMaxWidth(),
+            colors = ButtonDefaults.buttonColors(
```

Comment:
Again -  I think no need to set the colors here. I think it is already the default for the outlined button.